lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -684,6 +685,9 @@ def main():
         ret = None
         # First take care of creating the VM, if needed:
         if state in ('present', 'detached', 'attached'):
+            # Always activate disk when its being created
```

Comment:
You need to define vm_service:
```
vm_service = get_vm_service(connection, module)
```

Reply:
Yeah sorry it was defined already in devel i have not noticed that it wasnt in stable28